ublox-short-range/src/command/data_mode/types.rs

```diff
@@ -56,13 +56,13 @@ pub enum ServerType {
     #[at_arg(value = 2)]
     UDP(u16, UDPBehaviour, IPVersion),
     #[at_arg(value = 3)]
-    SSP(#[at_arg(len = 15)] String<128usize>),
+    SSP(#[at_arg(len = 15)] String<128_usize>),
     #[at_arg(value = 4)]
-    DUN(#[at_arg(len = 15)] String<128usize>),
+    DUN(#[at_arg(len = 15)] String<128_usize>),
```

Comment:
In all these cases where you are replacing from a `&str` with an `at_arg(len = ..)` annotation, you can make the new heapless string have that length.. Eg in this case it should end up as `String<15>`, with no `at_arg` annotation

Reply:
TIL! I fixed them.